dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ClientIT.java

```diff
@@ -257,7 +257,7 @@ public void testDepsCommandWithUnknownRunners() throws IOException {
     @Test
     public void testDepsCommand() throws IOException {
         Client.main(new String[] { "--config", TestUtility.getConfigFileLocation(true), "deps" });
-        Assert.assertTrue(systemOutRule.getLog().contains("avro=="));
+        Assert.assertTrue(systemOutRule.getLog().contains("monotonic=="));
```

Comment:
Actually, confused why this is assertTrue but the above is assertFalse

Reply:
The previous one is python 3 and this is python2. monotonic is the difference

Reply:
Oh, I see. With Python3 going to be the default, let's change `testDepsCommand` to `testDepsCommandWithVersionAndPython2` we should probably also update `scripts/install-tests.sh` but that can be a different PR